README.md

```diff
@@ -11,9 +13,18 @@ during an incident. Even though our users can benefit from having
 more diagnostics data coming out of services already instrumented
 with OpenCensus, they have to modify their code to register an
 exporter and redeploy. Asking our users recompile and redeploy is
-not an ideal at an incident time.
+not an ideal at an incident time. In addition, currently users need
+to decide which service backend they want to export to, before they
+distribute their binary instrumented by OpenCensus.
 
-OpenCensus service is trying to eliminate this requirement.
+OpenCensus Service is trying to eliminate these requirements. With
+OpenCensus Service, users do not need to redeploy or restart their binary
+as long as it has the OpenCensus Agent exporter. All they need to do is
+just configuring and deploying OpenCensus Service, then it will automatically
```

Comment:
just configure and deploy the OpenCensus service separately. OpenCensus service will then
automatically collect traces...